src/main/java/de/fraunhofer/aisec/cpg/passes/EvaluationOrderGraphPass.java

```diff
@@ -781,7 +782,10 @@ private void handleDoStatement(@NonNull Node node) {
     createEOG(doStatement.getStatement());
 
     createEOG(doStatement.getCondition());
+
+    doStatement.addPrevDFG(doStatement.getCondition());
```

Comment:
I am semantically confused about an evaluation order pass, that adds data flow edges. Does that make sense?